File: app/src/main/java/life/plank/juna/zone/data/network/model/LiveScoreData.java
package life.plank.juna.zone.data.network.model;

import lombok.Data;

@Data
public class LiveScoreData {
    private String timeStatus;
    private int minute;
    private int extraMinute;
    private int homeGoals;
    private int awayGoals;
    private int homeTeamPenaltyScore;
    private int awayTeamPenaltyScore;
    private String halfTimeScore;
    private String fullTimeScore;
    private String extraTimeScore;

    public LiveScoreData(String timeStatus, int minute) {
        this.timeStatus = timeStatus;
        this.minute = minute;
    }
}

File: app/src/main/java/life/plank/juna/zone/data/model/LiveScoreData.kt
package life.plank.juna.zone.data.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class LiveScoreData(
        var timeStatus: String = "",
        var minute: Int = 0,
        var extraMinute: Int = 0,
        var homeGoals: Int = 0,
        var awayGoals: Int = 0,
        var homeTeamPenaltyScore: Int = 0,
        var awayTeamPenaltyScore: Int = 0,
        var halfTimeScore: String = "",
        var fullTimeScore: String = "",
        var extraTimeScore: String = ""
) : Parcelable